TourMate/TourMate/Frontend/Plan/Views/TransportationOptionsView/TransportationOptionsViewModel.swift

```diff
@@ -89,7 +90,7 @@ class TransportationOptionsViewModel: ObservableObject {
         }
     }
 
-    @ViewBuilder func makeTransportationOptionsCellView(_ result: RoutingResult) -> some View {
+    func makeTransportationOptionsCellView(_ result: RoutingResult) -> some View {
```

Comment:
Could we shift out this part as well? VMs shouldnt be touching UI elements based on MVC style architecture